src/olympia/migrations/891-remove-compat-app.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE `compat_totals` DROP COLUMN `app`;
+DELETE FROM `compat_totals`;
```

Comment:
Is that really intended? I don't see in the description that we have to delete all data.